src/components/CommentCard.jsx

```diff
@@ -12,12 +13,29 @@ function CommentCard({comment,setComments}) {
         })
         incrementCommentVoteCount(comment_id,vote)
         
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault()
+        deleteComment(comment.comment_id)
+        setComments((currComments)=>{
+            
+            const currentComment = comment.comment_id
+            
+            const filteredComments=currComments.filter((currcomment)=>currcomment.comment_id !== currentComment)
+                return filteredComments
+        })
```

Comment:
I'd swap the order of these around which would make it more optimistic.
Also think about how to deal with if the request failed.
It would remove the comment from state but on a refresh the comment would appear back.